### PR TITLE
Added feature to get map for the album

### PR DESCRIPTION
**Pull request by mhz-edu**
Used leaflet library

### CODE REVIEW

File: src/views/map.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          return response.json();
+        })
+        .then((album) => {
+          const titleDiv = document.getElementById('title');
+          const title = document.createElement('h2');
+          title.textContent = album.title;
+          titleDiv.appendChild(title);
+
+          return fetch(url + '/api/albums/' + albumId + '/images');
+        })
```
Comment (irenlian) [inline src/views/map.html]: Could be optimised using Promise.all

File: src/views/map.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        .then((response) => {
+          return response.json();
+        })
+        .then((points) => {
+          const middlePoint = getMiddlePoint(points);
+          const map = L.map('map').setView(
+            [middlePoint.lat, middlePoint.lon],
+            13
```
Comment (irenlian) [inline src/views/map.html]: I'd advice moving it to constant value above

**Review by irenlian — APPROVED** [review]

File: src/views/map.html (hunk 2 above)
Comment (mhz-edu) [reply src/views/map.html]: Updated with constant

File: src/views/map.html (hunk 1 above)
Comment (mhz-edu) [reply src/views/map.html]: Updated with two async functions

**Pull request merged**